### ISSUE TITLE
oscillation of loss while training

### ISSUE THREAD

**Issue by Body123** [MIC-DKFZ/nnUNet#2175]
![taining](https://github.com/MIC-DKFZ/nnUNet/assets/24945076/c87d5010-5e8c-416d-8a39-018cab5a4ff0)
While training there are oscillations in the validation loss clearly, can you help me why or in there any advice to do while training 
and does the variable (num_iteration in epoch ) have an effect on that?

**Comment by Body123** [comment]
is there any help

**Comment by Karol-G** [comment]
Hey,

The reported pseudo-dice during training looks promising even though it is not the best. However, this is most likely due to the difficulty of the task or because the dataset is relatively small. An oscillation of the loss is always normal and is larger in cases like the previously mentioned ones.

Best,
Karol
